integration-tests/pom.xml

```diff
@@ -294,6 +294,7 @@
                 <module>rest-client</module>
                 <module>resteasy-reactive-kotlin</module>
                 <module>rest-client-reactive</module>
+                <module>rest-client-reactive-http2</module>
```

Comment:
I know we should avoid adding a new integration test modules, but I think in this case it makes sense since I didn't find any other module where this feature (testing http2 both in server and client side) fits. 

Note that the module `rest-client-reactive` uses Vert.x routes, not the `quarkus-resteasy-reactive` extension.

Anyway, if you think there is other module that fits better, let me know.

Reply:
I think it's warranted